### DISCUSSION TITLE
GStreamer AppSrc and AppSink plugins

### DISCUSSION THREAD

**Discussion by Zof-R** [gircore/gir.core#883] [Q&A] [answered]
Is AppSrc and AppSink usage possible in gir.core's current state? and if it is where can some guidance or sample be found?

**Comment by badcel** [comment, marked as answer]
GStreamer is currently not a priority for me. Meaning I did not take an explicit look at it. It is generated according to the gir files. But it is possible that there are fundamental things missing. As I'm not experienced at all with GStreamer my focus is to create good GObject bindings and if this is done I will take a dedicated look at GStreamer.

Some issues regarding GStreamer:
- #785 
- #784 
- #758 
- #159 

If you have experiences with GStreamer and are willing to contribute I can support you on the C# side of things to get things merged.

In conclusion I can't tell you how usable GStreamer is as I simply don't know. But any feedback or contribution is very welcome.
